target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -313,6 +313,12 @@ rp-n53)
 	ucidef_set_led_netdev "eth" "Network" "$board:white:back" "eth0"
 	set_wifi_led "$board:blue:wifi"
 	;;
+rt-ac51u)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	ucidef_set_led_netdev "activity" "activity" "$board:blue:activity" eth0.1
+	ucidef_set_led_netdev "activity" "activity" "$board:blue:activity" eth0.2
+	ucidef_set_wifi_led "$board:blue:activity"
```

Comment:
> With this configuration, the LAN leds and WAN does turn on when ethernet is connected

Which isn't related to this code, since you don't have any WAN or LAN leds in your dts file which can be controlled by the trigger.